charts/datree-admission-webhook/values.yaml

```diff
@@ -35,6 +40,8 @@ datree:
   enforce: ""
   # The name of the cluster link for cluster name in your dashboard. (string ,optional)
   clusterName: ""
+  # How often should the scan run in hours. If the inital scan is performed it will happen at least 55 minutes after, if its not performed it will happen 5 minutes after. (int, optional)
```

Comment:
I think this comment is not accurate. In the current implementation the initial scan is always preformed.
Am I right?

Reply:
You are correct, forgot to remove it after previous changes, removed now thank you!